report/reporter.rb

```diff
@@ -105,14 +106,26 @@ def coverage_stack
       @stack_tested_count * 100 / @apis[:json].count
     end
 
-    def display_endpoint(api, flavour = nil)
-      return "- `#{api}`" if flavour.nil?
-
-      if (test = find_test(api, flavour))
-        "- [x] <span title='tested'> [`#{api}`](#{test[:file]}\#L#{test[:line]})</span>"
-      else
-        "- [ ] <span title='not tested'> `#{api}`</span>"
-      end
+    def display_table
+      @apis[:specification].map do |api|
+        name = api['name']
+        tested_stack = if (test = find_test(name, :stack))
+                         "✔ [🔗](#{test[:file]}\#L#{test[:line]})</li></ul>"
+                       elsif stack_apis.include?(api)
+                         '❌'
+                       else
+                         'Not Applicable'
+                       end
+        tested_serverless = if (test = find_test(name, :serverless))
+                              "✔ [🔗](#{test[:file]}\#L#{test[:line]})</li></ul>"
```

Comment:
nit: how about `[✅](...)`?

Reply:
Yes, that does look better, the "link" emoji was something I put along the way and it never convinced me, I changed it to this :+1: